src/components/checkbox/checkbox.spec.js

```diff
@@ -246,5 +247,96 @@ describe('mdCheckbox', function() {
       expect(isChecked(checkbox)).toBe(false);
       expect(checkbox.hasClass('ng-invalid')).toBe(true);
     });
+
+    describe('with the md-indeterminant attribute', function() {
+
+      it('should set md-indeterminant attr to true by default', function() {
+        var element = compileAndLink(
```

Comment:
Do you need the wrapper `div`? Would `var checkbox = compileAndLink('<md-checkbox ... </md-checkbox>');` work?